index.html

```diff
@@ -69,6 +69,40 @@
                 </div>
             </div>
         </div>
+
+        <div id="tools-container">
+            <span class="title">
+                Tools
+            </span>
+            <div id="tools-container">
+                <div id="output-container">
+                    <div id="button-container">
+                        <button id="train-btn">Train</button><button id="analyze-btn">Analyze</button>
+                    </div> 
+                    <div id="tools-output">
+                        <table>
+                            <tbody>
+                                <tr>
+                                    <td>Accuracy:</td><td><span id="accuracy"></span></td>

```

Comment:
Here is an example of DOM in the HTML file with id, here we are using the `<span>` with id `accuracy`. We can add value into this using the JS file as follows:
```js
function train() {
    function callback(output) {
        console.log(output);
        $id("accuracy").innerHTML = output;
    }
    sendRequest(URL_TRAIN, {}, callback);
}
```
Hence the output of the HTTP request on URL_TRAIN will be handled by the `callback` function by changing the `innerHTML` of the `<span>` with id `accuracy`.

Reply:
See comment on line 107 of the `functions.js` file.